draft-regext-rdap-extensions.md

```diff
@@ -210,6 +219,82 @@ a mixed-case version of an existing identifier (see (#rdap_extensions_registry))
+with their RDAP extension identifier, instead using the extension
+identifier as the JSON name for that JSON value. That is, the
+extension identifier is used "bare" and not appended with an
+underscore character and subsequent names.
+
+Consider the example in (#child_json_values). Using the bare extension
+identifier pattern, that example could be written as:
+
+    {
+      "rdapConformance": [
+        "rdap_level_0",
+        "lunarNIC"
+      ],
+      "objectClassName": "domain",
+      "ldhName": "example.com",
+      "remarks":
+      [
+        {
+          "description":
+          [
+            "She sells sea shells down by the sea shore.",
+            "Originally written by Terry Sullivan."
+          ]
+        }
+      ],
+      "lunarNIC":
+      {
+        "firstInitial": "R",
+        "lastName": "Heinlein"
+      }
+    }
+
+A> The working group has not come to census on this topic. Therefore,
+A> three positions are presented.
+
+**BARE EXTENSIONS NOT ALLOWED**
+
+Usage of a bare extension identifier conflicts with the guidance in
+[@!RFC9083, section 2.1]. Previously, extension authors have used this
+pattern when only one query path, JSON name, or object class is being
+defined by the extension.
+
+Implementation experience has shown that an extension using a bare identifier can be
+interoperable though more difficult to process and parse in some instances.
+Furthermore, bare identifier's blur the line between what can be interpreted
+as an extension to RDAP vs core RDAP mechanisms.
+
+Henceforth, this pattern MUST NOT be used.
+
+**COMPROMISE: BARE EXTENSIOSN ONLY ALLOWED UNDER CERTAIN CONDITIONS**
+
+Usage of a bare extension identifier contravenes the guidance in [@!RFC9083, section 2.1].
+Implementation experience has shown that an extension using a bare identifier can be
+interoperable though more difficult to process and parse in some instances.
+Furthermore, bare identifier's blur the line between what can be interpreted
+as an extension to RDAP vs core RDAP mechanisms.
+
+This document updates [@!RFC9083] to explicitly allow this pattern only in IETF
+defined RDAP extensions and only when a technical solution cannot otherwise be defined.
+
+Along similar lines, an extension may define a single new object class,
+and use the extension's identifier as the object class name only in IETF defined
+RDAP extensions and only when a technical solution cannot otherwise be defined.
+
+**BARE EXTENSIONS ALLOWED**
+
+Usage of a bare extension identifier contravenes the guidance in [@!RFC9083, section 2.1].
+Implementation experience has shown that an extension using a bare identifier can be
+interoperable though more difficult to process and parse in some instances,
+therefore this document updates [@!RFC9083] to explicitly allow this pattern.
+
+Along similar lines, an extension may define a single new object class, and use the extension's identifier as the object class name.
```

Comment:
Isn't this redundant given we don't mention other naming scenarios in the preceding para?